content/docs/user-guide/gems/reference/multiplayer/multiplayer-gem/spawning.md

```diff
@@ -0,0 +1,35 @@
+---
+title: Spawning Autonomous Player Entities
+description: A reference for spawning and registering network player entities
+linktitle: Spawning Players
+---
+
+When a Player join a Multiplayer session, one of the first things that often occurs is spawning a networked entity for them to control. In order to do this, the Multiplayer Gem provides an interface to specify an entity for a Player and have the Gem mark it as Autonomous (as defined in (./overview)).
```

Comment:
+1 Love docs that start with a summary of "What can I achieve with this knowledge?"